DefensiveItem.h

```diff
@@ -2,6 +2,7 @@
 
 #include "Item.h"
 #include "Character.h"
+#include "Utility.h"
```

Comment:
Don't include what you don't use
no member functions or member variables in `Character` are being used in this header file
- [ ] revise